LiteCore/Query/SQLiteFleeceUtil.cc

```diff
@@ -150,7 +156,9 @@ namespace litecore {
                 case kNumber:
                     if ( val->isInteger() )
                         if ( val->isUnsigned() ) {
-                            sqlite3_result_int64(ctx, val->asUnsigned());
+                            // val->asUnsigned() does cause some overflow error with narrow_cast, so we can't use it
```

Comment:
Actually in release builds it does not, but this is fine as well

Reply:
😮